src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { getThaAwwz, currentAww, selectAww } from './actions'
+import Awwz from './Awwz.js'
+import BigAww from './BigAww.js'
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.getMoar = this.getMoar.bind(this)
```

Comment:
we're using https://github.com/jeffmo/es-class-fields-and-static-properties (https://babeljs.io/docs/plugins/transform-class-properties/) so you can do

```
getMoar = () => { }
```